tests/src/aurora_locker_utils.rs

```diff
@@ -172,6 +132,29 @@ impl Constructor {
     }
 }
 
+async fn forge_build(
+    libraries: &[String],
+    contract_output_path: &[&str],
+) -> anyhow::Result<serde_json::Value> {
+    let mutex_ref = &FORGE_LOCK;
+    let _guard = mutex_ref.lock().await;
```

Comment:
Out of curiosity, why do you need this mutex_ref, rather than call lock directly on the global variable?